src/git.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::Path;
+
+use git2::Repository;
+
+pub fn git_fetch(root: &Path) -> Result<(), git2::Error> {
+    let repo = Repository::open(root)?;
+    let mut remote = repo
+        .find_remote("origin")
+        .expect("Failed to find remote 'origin'");
```

Comment:
I guess these shouldn't happen in practice, but it would be good to return those errors rather than crash.

Reply:
Thanks, fixed that